packages/components/src/components/drawer/drawer.scss

```
@@ -51,12 +56,40 @@
             transition: 0.25s ease;
             transform: rotate(180deg);
         }
+        &-icon-mobile {
+            width: 2.4rem;
+            height: 0.8rem;
+            transition: 0.25s ease;
+            transform: rotate(0);
+        }
         &--open > &-icon {
             transform: rotate(0);
         }
+        &--open > &-icon-mobile {
+            transform: rotate(180deg);
+        }
+        &--mobile {
+            position: unset;
+            width: 100%;
+            height: 32px;
```

(suggest changing px to rem) 
```suggestion
            height: 3.2rem;
```